resources/documentation/roles.rst

```diff
@@ -7,11 +7,14 @@ This part is an agregations of all the **BlueBanquise** roles readme.
 CORE
 ====
 
+.. include:: roles/core/access_control/readme.rst
```

Comment:
while relevant, these changes are not related to monitoring. A separate PR would have been appropriate.

Reply:
My bad, I did it on the wrong branch. 🙁 

Lets keep it this way this time.